airflow/contrib/operators/bigquery_operator.py

```diff
@@ -45,6 +45,30 @@ def get_link(self, operator, dttm):
             job_id=job_id) if job_id else ''
 
 
+class BigQueryConsoleIndexableLink(BaseOperatorLink):
+    """
+    Helper class for constructing BigQuery link.
+    """
+
+    def __init__(self, index) -> None:
+        super().__init__()
+        self.index = index
+
+    @property
+    def name(self) -> str:
+        return 'BigQuery Console #{index}'.format(index=self.index + 1)
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        job_ids = ti.xcom_pull(task_ids=operator.task_id, key='job_id')
+        if not job_ids:
+            return None
+        if len(job_ids) < self.index:
+            return None
+        job_id = job_ids[self.index]
+        return 'https://console.cloud.google.com/bigquery?j={job_id}'.format(job_id=job_id)
```

Comment:
Can we extract this string to a template and reuse it in line 44?

Reply:
Updated. :-D